File: Level1/숫자 문자열과 영단어.java
class Solution {
    private String[] words = {"zero", "one", "two", "three", "four", "five",
            "six", "seven", "eight", "nine"};

    public int solution(String s) {
        for (int i = 0; i < 10; i++)
            s = s.replaceAll(words[i], Integer.toString(i));
        return Integer.parseInt(s);
    }
}

File: programmers-coding-test/C++/Level1/숫자 문자열과 영단어.cpp
#include <string>
#include <regex>

using namespace std;

int solution(string s) {
	string words[10] = { "zero", "one", "two", "three", "four", 
		"five", "six", "seven", "eight", "nine" };
	
	for (int i = 0; i < 10; i++)
		s = regex_replace(s, regex(words[i]), to_string(i));

	return stoi(s);
}